IFC4x3/Properties/s/SupportDescription_01gbdSFfjCmwU9fHFwjDV5/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Also seen this somewhere already.

Reply:
Can't find in the current spec?

Reply:
In that case, I merge and we try on the merged spec afterwards.